src/style.css

```diff
@@ -1,3 +1,103 @@
-body {
+head {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead, you can use a class or an ID for the `head` element if you need to style it. However, typically the `head` element does not contain styles as it's not rendered.